docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -210,7 +210,7 @@ Thus, Deckhouse images will be available at `https://your-harbor.com/d8s/deckhou
 Download script on a host that have access to `registry.deckhouse.io`. `Docker`, `crane` and `jq` must be installed on this host.
 
 ```shell
-curl -fsSL -o d8-pull.sh https://raw.githubusercontent.com/deckhouse/deckhouse/main/tools/release/d8-pull.sh
+curl -fsSL -o d8-pull.sh https://raw.githubusercontent.com/deckhouse/deckhouse/<Deckhouse version, for example, "v1.45.6">/tools/release/d8-pull.sh
```

Comment:
Why not something like this? Use this to get latest
```
export D8_RELEASE=$(curl -sL https://api.github.com/repos/deckhouse/deckhouse/releases | jq '.[0] | .tag_name' -r)
```
Or check for desired version https://github.com/deckhouse/deckhouse/releases and set it in
```
export D8_RELEASE="v1.45.6"
```
And then
```
curl -fsSL -o d8-pull.sh "https://raw.githubusercontent.com/deckhouse/deckhouse/$D8_RELEASE/tools/release/d8-pull.sh"
```

Reply:
We discussed that you couldn't automatically get the version. The script does not involve running on a master node or a node with kubectl configured, or the user needs to use some specific version.
It would be better to let the user think about which version to upload to his registry. 

My suggestion:
Download the script on a host that has access to the `registry.deckhouse.io` by specifying the Deckhouse release you need (e.g., `v1.45.6):

```
export D8_RELEASE="<DECKHOUSE_RELEASE>"
curl -fsSL -o d8-pull.sh https://raw.githubusercontent.com/deckhouse/deckhouse/${D8_RELEASE}/tools/release/d8-pull.sh
```

---
...or, avoid the variable:

...Download the script on a host that has access to the `registry.deckhouse.io` by specifying the Deckhouse release you need (e.g., `v1.45.6):

```
curl -fsSL -o d8-pull.sh https://raw.githubusercontent.com/deckhouse/deckhouse/<DECKHOUSE_RELEASE>/tools/release/d8-pull.sh
```

Reply:
@konstantin-axenov ^^

Reply:
Completely refactored